core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -1552,9 +1554,15 @@ public Expression implementResult(AggContext info,
       Expression acc = add.accumulator().get(0);
       Expression arg = add.arguments().get(0);
       SqlAggFunction aggregation = info.aggregation();
-      final Method method = (aggregation == BIT_AND
-          ? BuiltInMethod.BIT_AND
-          : BuiltInMethod.BIT_OR).method;
+
+      BuiltInMethod builtInMethod = BuiltInMethod.BIT_AND;
+      if (aggregation == BIT_OR) {
+        builtInMethod = BuiltInMethod.BIT_OR;
+      } else if (aggregation == BIT_XOR) {
+        builtInMethod = BuiltInMethod.BIT_XOR;
+      }
```

Comment:
Shall we do it like be below ?
```
if (aggregation  == BIT_AND) {
  ...
} else if (BIT_OR) {
  ...
} else if (BIT_XOR) {
  ...
} else {
  throw IllegalArgumentException
}
```
Or `Switch` is also cool for me.

Reply:
Thanks for your advise @jinxing64, I have updated.

Reply:
Use switch case seems more clear.

Reply:
@danny0405 Thank you, I have updated.

Reply:
@danny0405 Thank you, I have updated.